db/structure.sql

```diff
@@ -1553,16 +1245,12 @@ CREATE TABLE `transaction_transitions` (
   `transaction_id` int(11) DEFAULT NULL,
   `created_at` datetime DEFAULT NULL,
   `updated_at` datetime DEFAULT NULL,
+  `most_recent` tinyint(1) DEFAULT NULL,
```

Comment:
Same here, is this left here by accident, or is this required for the Rails 5 update?